cli/monkey-cam.cpp

```diff
@@ -97,10 +97,15 @@ std::unique_ptr<BoardShape> loadBoard(boost::property_tree::ptree& config) {
     tailPack = loadInserts(tpc.get());
   }
 
+  auto noseEdgeExt = config.get_optional<double>("board.nose edge extension")
+    .get_value_or(999.0);
```

Comment:
I really like your suggestion here but I'm struggling get it to work. I'm sure it shouldn't be too hard but my lack of C++ experience is failing me. It's falling over on the if statement - I think it is something to do with the MCFixed type but I'm a bit out of my depth...

I looked at what you've done for the refStance to make this optional for skis vs boards but this effectively uses a magic number of 0.0 (e.g. line 202 in shapes.cpp `if (m_refStance > 0.0) {...`). I chose 999 as 0 is a reasonable value for the edge extension...